plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/GenericConstants.java

```diff
@@ -45,6 +45,8 @@ public class GenericConstants {
     public static final String PARAM_OMIT_SINGLE_CATALOG = "omit-single-catalog";
     public static final String PARAM_OMIT_SINGLE_SCHEMA = "omit-single-schema";
     public static final String PARAM_OMIT_CATALOG_NAME = "omit-catalog-name";
+    public static final String PARAM_STORES_UNQUOTED_CASE = "stores_unquoted_case";
+    public static final String PARAM_STORES_QUOTED_CASE = "stores_quoted_case";

```

Comment:
Use `-` instead of `_` to be consistent with other parameters.

```suggestion
    public static final String PARAM_STORES_UNQUOTED_CASE = "stores-unquoted-case";
    public static final String PARAM_STORES_QUOTED_CASE = "stores-quoted-case";

```